app.js

```diff
@@ -78,9 +87,59 @@ const STREAM_WORST_QUALITY_QUERY =
   '[data-a-target="player-settings-menu"] .tw-pd-05:last-child .tw-radio';
 const CHANNEL_STATUS_QUERY = ".tw-channel-status-text-indicator";
 const NO_DROPS_QUERY = 'div[data-test-selector="drops-list__no-drops-default"]';
+const DROP_INVENTORY_LIST = 'div.tw-flex-wrap.tw-tower.tw-tower--180.tw-tower--gutter-sm';
+const DROP_ITEM = '.tw-flex';
+const CATEGORY_NOT_FOUND = '[data-a-target="core-error-message"]';
+const DROP_STATUS = '[data-a-target="Drops Enabled"]';
 
 // ========================================== CONFIG SECTION =================================================================
 
+// ========================================== UTILS SECTION =================================================================
+/**
+* @fn idle
+* @param (ms) number of milliseconds to wait until the program resumes execution 
+* @author AlexSimpler
+* @return a promise which resolves after a timeout
+*/
+function idle(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+/**
+* @fn upperFirst
+* @param (word) makes the first letter of the word uppercase
+* @author AlexSimpler
+* @return the modified word
+*/
+function upperFirst(word) {
+  return (word[0].toUpperCase() + word.substring(1));
+}
+
+/**
+* @fn getUserProperty (Beta)
+* @param (page, name) takes as parameters the page handler aswell as the name of the property inside twilight-user
+* @author AlexSimpler
+* @return the value of the cookie
+*/
+async function getUserProperty(page, name) {
+
+  if (!name || !(/^[A-Za-z1-9]+$/.test(name))) throw new Error("Invalid cookie name: ", name);
+
+  const data = await page.cookies();
+  let cookieValue = undefined;
```

Comment:
We can simplify this code by using `Array.prototype.find`

Reply:
Ok, good point, you can simplify it.